packages/product-react/src/components/Input/Input.tsx

```diff
@@ -0,0 +1,112 @@
+import clsx from 'clsx';
+import React from 'react';
+import FieldWrapper from '../Common/FieldWrapper';
+import FormItemWrapper from '../Common/FormItemWrapper';
+import IconWrapper from '../Common/IconWrapper';
+import RTLWrapper from '../Common/RTLWrapper';
+
+export enum IconPosition {
+  Prepend = 'PREPEND',
+  End = 'END',
+  Start = 'START'
```

Comment:
nit these should be all UPPERCASE https://github.com/airbnb/javascript#naming--uppercase

Reply:
uhmm good to know, but was following this convention, https://www.typescriptlang.org/docs/handbook/enums.html#string-enums, what do you think ?

Reply:
oo that's interesting. I've always used uppercase in both JS and python. a nit though so up to you!